test/controllers/ChargeSummaryControllerSpec.scala

```diff
@@ -185,6 +185,26 @@ class ChargeSummaryControllerSpec extends ChargeSummaryControllerHelper {
                 document.getElementsByClass("govuk-heading-l").first().text() shouldBe "Interest on your late submission penalty"
                 document.getElementById("payment-history-table").getElementsByTag("caption").text() shouldBe "Late submission penalty history"
               }
+
+              "provided with an id associated to a Late payment penalty" in new Setup(testValidFinancialDetailsModelWithLatePaymentPenalty){
+                enable(ReviewAndReconcilePoa)
```

Comment:
we already have a version of enable where we can pass a variable number of arguments